### DISCUSSION TITLE
卡在创建一个项目上了

### DISCUSSION THREAD

**Discussion by lzkids** [xaoxuu/hexo-theme-stellar#82] [General]
完善项目信息填在projects.yml里，但是多个项目怎么完善？需要建立多个projects.yml吗？项目文档标签建立之后，怎么让项目文档标签生效。我在md添加了tags但是不起作用。点击之后里面是空白的，我看路径是`/wiki/tags/%E7%AC%94%E8%AE%B0/index.html`

**Comment by xaoxuu** [comment]
https://github.com/xaoxuu/xaoxuu.github.io/blob/main/source/_data/projects.yml
